src/main/java/ru/rosatom/documentflow/controllers/DocumentController.java

```diff
@@ -204,4 +203,23 @@ public List<DocumentChangesDto> findDocChangesByUserId(
                 .map(o -> modelMapper.map(o, DocumentChangesDto.class))
                 .collect(Collectors.toList());
     }
+
+    @Operation(summary = "Оставить комментарий к к документу")
+    @PatchMapping("/addComment/{documentId}")
+    @SecurityRequirement(name = "JWT")
+    @PreAuthorize("@documentProcessSecurityService.isMyCompany(#documentId,#user.id) && " +
+            "(hasAuthority('USER') || hasAuthority('COMPANY_ADMIN'))")
+    public DocumentDto newComment(
+            @PathVariable @Parameter(description = "ID документа") Long documentId,
+            @RequestBody @Valid @Parameter(description = "Текст нового комментария") String content,
+            @AuthenticationPrincipal @Parameter(description = "Пользователь", hidden = true) User user) {
+        log.trace("Добавлен новый комментарий к документу {} от пользователя {}", documentId, user.getId());
+        DocProcessComment comment = documentProcessService.createComment(content, user, null );
```

Comment:
Тут наверное лучше debug уровень логирования
```suggestion
        log.debug("Добавлен новый комментарий к документу {} от пользователя {}", documentId, user.getId());
        DocProcessComment comment = documentProcessService.createComment(content, user, null );
```

---

Передача null в качестве параметров не очень хороший тон. Используй перегрузку метода чтобы убрать null